fighters/shulk/src/opff.rs

```diff
@@ -68,11 +51,44 @@ unsafe fn fastfall_specials(fighter: &mut L2CFighterCommon) {
     }
 }
 
+unsafe fn arts_cancelling(fighter: &mut L2CFighterCommon, status_kind: i32) {
+    if fighter.is_motion_one_of(&[
+        Hash40::new("attack_13"),
+        Hash40::new("attack_s3_s"),
+        Hash40::new("attack_hi3"),
+        Hash40::new("attack_lw3"),
+        Hash40::new("attack_air_n"),
+        Hash40::new("attack_air_f"),
+        Hash40::new("attack_air_b"),
+        Hash40::new("attack_air_hi"),
+        Hash40::new("attack_air_lw") ]) {
+        if AttackModule::is_infliction_status(fighter.module_accessor, *COLLISION_KIND_MASK_HIT | *COLLISION_KIND_MASK_SHIELD) {
+            println!("beat");
+            VarModule::on_flag(fighter.object(), vars::shulk::instance::MONADO_BEAT);
```

Comment:
Please make this a status variable instead, and remove the second half of the function that turns off the flag

Reply:
oh and the println I guess

Reply:
done. however, making this a status variable makes the effects on a successful arts cancel not play.

Reply:
That can probably be resolved via modifying the monado art activation pre status and setting the variable again